features/admin.tenants.v1/models/tenants.ts

```diff
@@ -158,3 +158,50 @@ export interface TenantListResponse {
      */
     tenants: Tenant[];
 }
+
+/**
+ * Represents the response structure for retrieving details of the
+ * currently authenticated organization.
+ */
+export interface OrganizationSelfResponse {
+    /**
+     * The unique identifier of the organization.
+     */
+    id: string;
+    /**
+     * The name of the organization.
+     */
+    name: string;
+    /**
+     * The organization handle.
+     */
+    orgHandle: string;
+    /**
+     * The description of the organization.
+     */
+    description: string;
+    /**
+     * The status of the organization.
+     */
+    status: string;
+    /**
+     * The version of the organization.
+     */
+    version: string;
+    /**
+     * The date when the organization was created.
+     */
+    created: string;
+    /**
+     * The date when the organization was last modified.
+     */
+    lastModified: string;
+    /**
+     * The type of the organization.
+     */
+    type: string;
```

Comment:
If the `type` can hold a set of specific values, it would be better to have an enum defined for those options to enhance readability.